core/config.class.inc.php

```diff
@@ -1320,6 +1320,14 @@ class Config
 			'source_of_value' => '',
 			'show_in_conf_sample' => false,
 		],
+		'security_header_xcontent_type_options' => [
+			'type' => 'bool',
+			'description' => 'If set to false, iTop will stop sending the X-Content-Type-Options HTTP header.',
+			'default' => 'true',
```

Comment:
```suggestion
		'security.enable_header_xcontent_type_options' => [
			'type' => 'bool',
			'description' => 'If set to false, iTop will stop sending the X-Content-Type-Options HTTP header.',
			'default' => true,
```

- Namespace the conf. param. like for all new params.
- Fix default value to bool instead of string.
- Maybe we could explain in the description for which reasons it could be good to switch it to false?

Reply:
Argll good catch for the default value :(
I changed also the description
Done in 5a1dd9ff

We'll have a chat on renaming

Reply:
Renaming done in c630a3f1

Reply:
Woops renamed done in da941fca actually (missed the enable prefix)